shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java

```diff
@@ -57,6 +58,7 @@ public void prepareDependencies() throws IOException {
     mMetaMasterClient = mock(MetaMasterClient.class);
     MasterInfo masterInfo = MasterInfo.newBuilder()
         .setLeaderMasterAddress("testAddress")
+        .addMasterAddresses(NetAddress.newBuilder().setHost("testAddress2").build())
```

Comment:
Can we test multiple master addresses, like 3 masters here including the leader master address?

Reply:
The master addresses should include the leader master address and are better to include three master net addresses. Three is a typical number to achieve fault tolerance.